src/styles/partials/components/_walking-trails.scss

```diff
@@ -269,11 +306,25 @@
       margin: $default-margin;
       width: 100%;
     }
+
+    .search-container-trails {
+      .search-container_inner {
+        padding: 0 $padding-large;
+      }
+    }
   }
 }
 
 @media (max-width: 600px) {
-  .js-trails-filter {
-    width: 100%;
+  .filter-walking-trails {
```

Comment:
Do we have a variable for the 600px? If not then ignore this comment

Reply:
We don't have one for that number, but I was able to test if the next closest breakpoint we had a variable for would work, and everything still looks ok. So I think it's a bit cleaner now and still works.